Providers/Profile.swift

```diff
@@ -1089,12 +1027,6 @@ open class BrowserProfile: Profile {
             }
         }
 
-        fileprivate func syncLegacyHistoryWithDelegate(_ delegate: SyncDelegate, prefs: Prefs, ready: Ready, why: SyncReason) -> SyncResult {
-            log.debug("Syncing history to storage.")
-            let historySynchronizer = ready.synchronizer(HistorySynchronizer.self, delegate: delegate, prefs: prefs, why: why)
-            return historySynchronizer.synchronizeLocalHistory(self.profile.history, withServer: ready.client, info: ready.info, greenLight: self.greenLight())
```

Comment:
`func greenLight()` should be removed since not used anymore by removing this code